docs/preview/features/secret-store/index.md

```diff
@@ -78,6 +77,34 @@ public class HealthController : ControllerBase
 }
 ```
 
+### Using secret store outside .NET hosting
+The secret store is also available directly on the `IServiceCollection` for applications that run outside the .NET hosting but still want to use the Arcus secret store.
+
+Just like you would register the secret store on the `HostBuilder`, you can use the `.AddSecretStore` extension to register the secret store:
+
+```csharp
+IServiceCollection services = ...
+IConfiguration configuration = ...
+
+// Mark: the secret store uses logging, so make sure that you have set the logging beforehand.
+service.AddLogging(...);
+
+services.AddSecretStore(stores =>
```

Comment:
`AddSecretStore` or `AddSecretProviders` ?

Reply:
Let's stick with `AddSecretStore` to align with our `ConfigureSecretStore` unless you want to use `ConfigureSecretStore` here as well; but typically services use `Add` as far as I know

Reply:
Yes, agree, keeping within the .NET naming.